utils/R/read.output.R

```diff
@@ -123,8 +123,6 @@ read.output <- function(runid, outdir, start.year=NA,
 
   if(!nofiles){
     for(ncfile in ncfiles) {
-      valid.nc <- tryl(nc_open(ncfile))
```

Comment:
@tonygardella this line was causing too many open files issue, so my fix is to delete it and remove the if-block around the code that opens and reads the ouput-netcdf file. I'm not sure if it will also make the workflow run faster but at least too many open files won't be an issue regarding this bit of code